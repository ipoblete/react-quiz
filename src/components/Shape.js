import React from 'react';
import PropTypes from 'prop-types';

function Shape({ text, fontColor, backgroundColor }) {
  return (
    <div style={{ width: '100px', height: '100px', color: fontColor, backgroundColor }}>
      {text}
    </div>
  );
}

Shape.propTypes = {
  text: PropTypes.string.isRequired,
  fontColor: PropTypes.string.isRequired,
  backgroundColor: PropTypes.string.isRequired
};

export default Shape;
