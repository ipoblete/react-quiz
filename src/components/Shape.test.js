import React from 'react';
import renderer from 'react-test-renderer';
import Shape from './Shape';

describe('Shape', () => {
  it('matches a snapshot', () => {
    const shape = renderer.create(
      <Shape />
    ).toJSON();
    expect(shape).toMatchSnapshot();
  });
});
