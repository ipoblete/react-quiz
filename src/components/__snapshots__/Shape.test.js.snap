// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shape matches a snapshot 1`] = `
<div
  style={
    Object {
      "backgroundColor": undefined,
      "color": undefined,
      "height": "100px",
      "width": "100px",
    }
  }
/>
`;
